Description: remove windows dependencies from Cargo.toml
Author: Nadzeya Hutsko <nadzya.info@gmail.com>
Forwarded: not-needed
Last-Update: 2025-09-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -70,8 +70,3 @@
 
 [dependencies.termcolor]
 version = "1"
-
-[target."cfg(windows)".dependencies.crossterm]
-version = "0.28"
-features = ["windows"]
-default-features = false
